stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Android.java

```diff
@@ -22,4 +26,20 @@ public void bindingWithViewModel() {
             ChannelListHeaderViewModelBinding.bind(viewModel, channelListHeaderView, getViewLifecycleOwner());
         }
     }
+
+    /**
```

Comment:
This link is incorrect
@see <a href="https://getstream.io/chat/docs/android/channel_list_header_view">Channel List Header View</a>